packages/forklift-console-plugin/src/modules/Plans/views/list/components/PlanStatusCell.tsx

```diff
@@ -45,8 +45,7 @@ export const PlanStatusCell: React.FC<CellProps> = ({ data }) => {
   const [lastMigration] = usePlanMigration(plan);
   const [isButtonEnabled, setIsButtonEnabled] = useState(true);
 
-  const isWarmAndExecuting = plan.spec?.warm && isPlanExecuting(plan);
-  const isWaitingForCutover = isWarmAndExecuting && !isPlanArchived(plan);
+  const isWarmAndExecuting = plan.spec?.warm && isPlanExecuting(plan) && !isPlanArchived(plan);
```

Comment:
This part also repeat itself a few times, consider extracting it to a const/util func